xl-op/digitalai/dai-deploy/kubernetes-generic/dai-deploy_cr_default.yaml.tmpl

```diff
@@ -252,8 +254,6 @@ spec:
       repository: {{ .RepositoryName }}/postgresql
     {{- end }}
     install: {{ .EnablePostgresql }}
-    service: 
```

Comment:
Should we not have this field. It is just duplicate of `postgresql.global.postgresql.service.ports` and `postgresql.primary.service.ports.postgresql`

Reply:
Yes, correct, because of that I am removing/revarting this conf, that was workarround for the release